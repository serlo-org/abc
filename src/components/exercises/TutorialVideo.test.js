import React from 'react';

import renderer from 'react-test-renderer';

jest.mock('expo', () => ({
  Video: 'Video'
}));

const TutorialVideo = require('./TutorialVideo').default;

it('renders without crashing', () => {
  const tree = renderer.create(
    <TutorialVideo video={require('../../assets/videos/placeholder.mp4')} />
  );
  expect(tree).toMatchSnapshot();
});
