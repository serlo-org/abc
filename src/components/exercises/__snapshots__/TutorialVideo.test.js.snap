// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Video
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      callback={[Function]}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      resizeMode={undefined}
      shouldPlay={true}
      source={1337}
      style={
        Object {
          "width": "100%",
        }
      }
      testID={undefined}
    />
  </View>
</View>
`;
